octobot_trading/personal_data/positions/position.py

```diff
@@ -273,7 +273,8 @@ def update_from_order(self, order):
         size_update = self._calculates_size_update_from_filled_order(order, size_to_close)
 
         # Updates position average entry price from order
-        self.update_average_entry_price(size_update, order.filled_price)
+        if self._is_update_increasing_size(size_update):
```

Comment:
:+1: Nice fix

Reply:
We can also do something like `if has_increase_position_size := self._is_update_increasing_size(size_update):` and remove `_update_size` return and `_update_size`'s `_is_update_increasing_size()` call.

Reply:
We can also do something like if has_increased_position_size := self._is_update_increasing_size(size_update): and remove _update_size return and _update_size's _is_update_increasing_size() call.

Reply:
right

Reply:
done in  https://github.com/Drakkar-Software/OctoBot-Trading/pull/682